initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -101,26 +101,17 @@ fi
 
 if [ -e /etc/debian_version ]; then
     echo "Removing DEB"
-    if [ $force -eq 0 ]; then
-        dpkg --purge hpccsystems-platform
-        if [ $? -ne 0 ]; then
-           message dpkg
-           exit 1
-        fi
-    else
-        dpkg --purge --force-all hpccsystems-platform
+    apt-get remove -y hpccsystems-platform
+    if [ $? -ne 0 ]; then
+       echo "An error has occured during removal.  Try \`apt-get -f remove hpccsystems-platform\` to fix this issue."
```

Comment:
I changed the error message to something more appropriate, now that we aren't using dpkg/rpm for removal.